cli/bufflib/src/artifact.rs

```diff
@@ -51,8 +50,8 @@ fn save_buffer_to_file(buffer: Vec<u8>, path: &str) {
 #[test]
 fn should_create_artifact() {
```

Comment:
i really love having unit tests so close to the code. makes everything so easy to grasp and maintain!